workflows/config/300-workflow.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: workflows.tekton.dev
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+    pipeline.tekton.dev/release: "devel"
+    version: "devel"
```

Comment:
im guessing we dont need `pipeline.tekton.dev/release`, im wondering about `version` as well 🤔 

(looks like [task-loops has both of those but im pretty sure the release tag at least isnt needed](https://github.com/tektoncd/experimental/blob/main/task-loops/config/300-taskloop.yaml)

Reply:
Removed the release label